app/controllers/event_session_comments_controller.rb

```diff
@@ -0,0 +1,44 @@
+class EventSessionCommentsController < ApplicationController
+  before_filter :authenticate_user!
+
+  def create
+    @event_session_comment = EventSessionComment.new(comment_params)
+
+    @event_session_comment.save
+
+    redirect_to controller: 'event_sessions', action: 'show', id: params[:event_session_comment][:event_session_id].to_i
+  end
+
+  def edit
+    @event_session_comment = EventSessionComment.find(params[:id])
+    @event_session = EventSession.find(@event_session_comment.event_session_id)
+
+    respond_to do |format|
+      format.html {redirect_to @event_session}
+      format.js {render 'event_sessions/event_session_comments/edit_comment'}
+    end
+  end
+
+  def update
+    @event_session_comment = EventSessionComment.find(params[:id])
+    @event_session = EventSession.find(@event_session_comment.event_session_id)
+
+    @event_session_comment.update(comment_params)
```

Comment:
Test for update to be successful and response accordingly. Check for example:

https://github.com/null-open-security-community/swachalit/blob/master/app/controllers/session_proposals_controller.rb